action.yaml

```diff
@@ -19,10 +19,6 @@ runs:
   using: "composite"
 
   steps:
-    - name: Change to working directory
```

Comment:
Agree with removing this, as it's not carried into the next step. 
Though `Upload Lambdas to S3` require a value for working-directory. So can't make field optional, at least not without a default.

Reply:
The `working-directory` can be passed an empty value - this is correct.
The param doesn't need to have a defaul of `.` - `working-directory` is null by default so passing null/empty has no effect, as desired.